scripts/script.js

```diff
@@ -55,11 +55,11 @@ class Script {
     if (!process.env.NO_SCRIPT) {
       Promise
         .resolve()
-        .then(() => this.db.open())
+        .then(() => this.setUp())
```

Comment:
Yes and no.

With `.then(this.setUp)` you are passing the function itself directly. It will be called by the promise chain without its receiver, i.e. `this` is not conserved (see https://codepen.io/AlphaHydrae/pen/wOZYeL?editors=0011).

With `.then(() => this.setUp())` or `.then(this.setUp.bind(this))`, then `this` will be conserved.

Reply:
Yeah, this damn `this` in arrow functions. Got it.